automatic/sqlite/tools/chocolateyInstall.ps1

```diff
@@ -1,23 +1,24 @@
 ﻿$ErrorActionPreference = 'Stop'
 
 $toolsDir = Split-Path $MyInvocation.MyCommand.Definition
-$embedded_path = if ((Get-OSArchitectureWidth 64) -and $env:chocolateyForceX86 -ne 'true') {
-         Write-Host "Installing 64 bit version"; Get-Item "$toolsDir\*dll*win64*.zip"
-} else { Write-Host "Installing 32 bit version"; Get-Item "$toolsDir\*dll*win32*.zip" }
 
 $packageArgs = @{
-    PackageName  = 'sqlite'
-    FileFullPath = $embedded_path
-    Destination  = $toolsDir
+    PackageName    = 'sqlite'
+    FileFullPath   = (Get-Item "$toolsDir\*dll*win-x86*.zip")
+    FileFullPath64 = (Get-Item "$toolsDir\*dll*win-x64*.zip")
+    Destination    = $toolsDir
 }
 Get-ChildItem $toolsDir\* | Where-Object { $_.PSISContainer } | Remove-Item -Recurse -Force #remove older package dirs
 Get-ChocolateyUnzip @packageArgs
 
 $pp = Get-PackageParameters
 if (!$pp.NoTools) {
+    if ((Get-OSArchitectureWidth -Compare 32) -or ($env:chocolateyForceX86 -eq 'true')) {
+        Write-Error -Message "32-bit version of sqlite tools not available after version 3.43.2" -Category ResourceUnavailable
```

Comment:
```suggestion
        Write-Error -Message "The 32-bit version of sqlite tools is not available after version 3.43.2" -Category ResourceUnavailable
```

Reply:
Updated error message to reflect suggested grammatical changes/proper English.